doxia-site-model/src/main/mdo/site.mdo

```diff
@@ -66,6 +66,15 @@ under the License.
            <type>String</type>
            <defaultValue>merge</defaultValue>
         </field>
+        <field xml.attribute="true">
+           <description><![CDATA[
+             Whether to enforce a parent site descriptor.
+           ]]></description>
+           <name>enforceParentDescriptor</name>
```

Comment:
not sure this is really a good name, any other suggestion?

Reply:
Instead of `enforce` I'd use the term 'requre`. E.g., `requireParentSite`. The term `descriptor` is too confusing and too abstract for most.